docs/get_started/contribute/community.rst

```diff
@@ -0,0 +1,96 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Is it possible to leave the folder sections of get_started as is for now 
- docs/contribute
- docs/install
The main reason are:
- These are links we commonly link from other places and change them will result in stale links
- We can keep essentially the same navigation structure.

Reply:
i'm not sure we should carve out exceptions just due to legacy. what about adding a redirect?

Reply:
a redirect also works